modules/sb-install-service-catalog.adoc

```diff
@@ -10,47 +10,38 @@ enable it using the procedure below.
 
 .Procedure
 
-. Create the service catalog API server.
-.. Create a file named `svcat-api-server.yaml` with the following contents.
+. Enable the service catalog API server.
+.. Use the following command to edit the service catalog API server resource.
 +
-[source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: ServiceCatalogAPIServer
-metadata:
-  name: cluster
-spec:
-  logLevel: "Normal"
-  managementState: Managed
+$ oc edit servicecatalogapiservers.operator.openshift.io cluster
```

Comment:
perhaps there is a standard format you are following here, if so, great.  Otherwise, this command can be abbreviated a bit if you wish to:
`oc edit servicecatlaogapiservers`